### PR TITLE
fix: milestone card ux updates

### PR DESCRIPTION
**Pull request by djmcquillan**
fixes: #273 

- Minor styling updates to the milestone cards

demo:
https://starmaps-9rwlhr85y-ipfs-ignite.vercel.app/roadmap/github.com/ipfs/go-libipfs/issues/54#detail

### CODE REVIEW

**Comment by djmcquillan** [comment]
@whizzzkid updated the github icon.  As for the hover state of the cards the hover border change should only occur if there are children for the issue

**Comment by SgtPooki** [comment]
@djmcquillan when is the github icon supposed to be muted? I would imagine it should be muted at all times unless hover over.. but I'm not sure what you and @juliaxbow talked about

File: components/roadmap-grid/grid-row.tsx [hunk 1]
```diff
@@ -85,20 +84,22 @@ export function GridRow({
...
     >
-      <Flex direction={{ base:"column", md:"column", lg:"row" }} justify="space-between" position="relative">
-        <Flex direction="column" maxW={{ base: "100%", sm: "100%", md:"100%", lg:"85%" }}>
-          <Text as="b" className={styles.milestoneTitleWrapper}>{milestone.title.value}</Text>
-          <p className={styles.milestoneDate}>{milestone.due_date.value}</p>
+      <Flex direction={{ base:"column" }} justify="space-between" position="relative" w="100%">
+        <Flex direction="column" w="100%">
+          <Text as="b" className={`${styles.milestoneTitleWrapper} ${clickable && styles.milestoneTitleWrapperLink}`}>{milestone.title.value}</Text>
+          <Flex h='8px' w="100%" borderRadius="20px" bgColor="#F1F4F8">
+            <Box w={`${parseInt(milestone.completion_rate.value.toString())}%`} h="100%" borderRadius="20px" bg="#7DE087" />
```
Comment (SgtPooki) [inline components/roadmap-grid/grid-row.tsx:95]: I see that this is existing code but I'm confused why are we converting a number to a string to an int to a string? 

```
w={`${parseInt(milestone.completion_rate.value.toString())}%`}
```

If there is some need to truncate, we should use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Number/toPrecision where we set `completion_rate`. Could you address that @djmcquillan ?

**Review by SgtPooki — CHANGES_REQUESTED** [review]
just one thing that isn't entirely your code

**Comment by djmcquillan** [comment]
> @djmcquillan when is the github icon supposed to be muted? I would imagine it should be muted at all times unless hover over.. but I'm not sure what you and @juliaxbow talked about

just based on the figma designs the github icon is always muted unless hovered - update was made to address the non-muted icons in latest commit.

**Comment by SgtPooki** [comment]
demo looks much better

**Review by SgtPooki — APPROVED** [review]

**Comment by djmcquillan** [comment]
> demo looks much better

@SgtPooki are you ok with merging these changes in before we address the comments in #273 ?

**Comment by SgtPooki** [comment]
Yessir. thanks for double checking.. we can adjust later if necessary! Sounds like julia did all the design work required and I was late to the party. sorry for the holdup

**Comment by SgtPooki** [comment]
ugh. accidentally clicked close instead of comment

**Pull request merged**